app/controllers/actors/ProvisionerProcessor.scala

```diff
@@ -1,13 +1,14 @@
 package controllers
 package actors
 
-import akka.util.Duration
+import scala.concurrent.duration._
 import collins.provisioning.ProvisionerRequest
 import collins.shell.CommandResult
 import com.twitter.util.Future
```

Comment:
I'm a bit confused to why we would be pulling in Twitter Futures/classes.  Is there any particular reason for this?